docs/guides/alert-manager-configuration.md

```diff
@@ -9,6 +9,23 @@ slug: alertmanager-configuration
 
 Cortex Alertmanager notification setup follow mostly the syntax of Prometheus Alertmanager since it is based on the same codebase.  The following is a description on how to load the configuration setup so that Alertmanager can use for notification when an alert event happened.
 
+### Configuring the Cortex Alertmanager storage backend
+
+With the introduction of Cortex 1.8 the storage backend config option shifted to the new pattern [#3888](https://github.com/cortexproject/cortex/pull/3888). For example the new configuration pattern for the s3 backend looks like this
+```YAML
+alertmanager_storage:
+  backend: "s3"
+  s3:
+    endpoint: <endpoint>
+    bucket_name: <bucket>
+    secret_access_key: <secret_access_key>
+    access_key_id: <access_key_id>
+```
+All blocks storage bucket clients and the config service are currently supported. If you choose to try the experimental Alertmanager resharding feature then please note that the local and configdb storage backends are not supported.
```

Comment:
> All blocks storage bucket clients and the config service are currently supported. 

I think this is redundant - I would re-work this to include a reference the auto-generated documentation; this is kept in sync with the code: https://cortexmetrics.io/docs/configuration/configuration-file/#alertmanager_storage_config

> If you choose to try the experimental Alertmanager resharding feature then please note that the local and configdb storage backends are not supported.

I would be more explicit here, for example:

> Note that when using `-alertmanager.sharding-enabled=true`, the following storage backends are not supported: `local`, `configdb`.

Reply:
Thank you so much. As you can probably tell english is not my first english but hey :D one can try

I agree with the redundant section and reworked it a bit and linked back to the configuration file. If you don't have anything more to add then I would like to rebase and squash my commits.